libportal/portal.c

```diff
@@ -380,14 +380,21 @@ gboolean
 xdp_portal_running_under_flatpak (void)
 {
   static gsize under_flatpak;
```

Comment:
```suggestion
  static gsize under_flatpak = 0;
```

because explicit is better than implicit?

Other than that, LGTM